docs/docs/spaces.mdx

```diff
@@ -0,0 +1,246 @@
+---
+id: spaces
+sidebar_label: Spaces
+title: Spaces
+description: Learn about Spaces for Rasa.
+abstract: Spaces are a way to modularize your assistant that increases isolation between
+subparts, and classification performance for intents and entities that are only
+relevant in specific contexts.
+---
+
+import RasaProLabel from "@theme/RasaProLabel";
+import RasaProBanner from "@theme/RasaProBanner";
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<RasaProLabel />
+
+<RasaProBanner />
+
+:::caution New in 3.6.0a1
+This feature is experimental.
+We introduce experimental features to get feedback from our community, so we encourage you to try it out!
```

Comment:
This alpha release will allow us to gather feedback from our users and integrate it into future releases. We encourage you to try this out!